modules/kotest/src/test/kotlin/org/testcontainers/containers/TestLifecycleAwareTest.kt

```diff
@@ -0,0 +1,17 @@
+package org.testcontainers.containers
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class TestLifecycleAwareListenerTest : StringSpec({
```

Comment:
1. the error case is not tested
2. the filename logic is not tested

Reply:
Added test for filename logic